### PR TITLE
Initial IonQ service

### PR DESCRIPTION
**Pull request by dabacon**
Contents
* Supports create_job and get_job
* Does some minimal serialization
#3479 has list of further features to be added.

### CODE REVIEW

File: cirq/ionq/ionq_client_test.py
Comment (balopat) [inline cirq/ionq/ionq_client_test.py:218]: :)

File: cirq/ionq/serializer.py
Comment (balopat) [inline cirq/ionq/serializer.py:41]:
```suggestion
            raise ValueError('Cannot serialize empty circuit.')
```

**Review by balopat — APPROVED** [review]
LGTM, welcome to Cirq IonQ package! 
Looking forward to the follow-up PRs to extend the serialization, the results, all the TODOs and all the documentation!
Also, it would be great if you could give me another user name so that we can add you two to the upcoming "owners" list - this way, you will be able to progress a bit faster without having to wait on Cirq maintainer reviews.

File: cirq/ionq/serializer.py
Comment (dabacon) [reply cirq/ionq/serializer.py]: this has been added.

**Comment by dabacon** [comment]
Do you want exactly two owners or more?

**Review by vtomole — APPROVED** [review]
LGTM

**Comment by balopat** [comment]
> Do you want exactly two owners can there be more?

There can be more!

**Pull request merged**
